src/machines/java/com/enderio/machines/client/gui/screen/PaintingMachineScreen.java

```diff
@@ -0,0 +1,49 @@
+package com.enderio.machines.client.gui.screen;
+
+import com.enderio.EnderIO;
+import com.enderio.api.misc.Vector2i;
+import com.enderio.base.common.lang.EIOLang;
+import com.enderio.core.client.gui.screen.EIOScreen;
+import com.enderio.core.client.gui.widgets.EnumIconWidget;
+import com.enderio.core.client.gui.widgets.ToggleImageButton;
+import com.enderio.machines.client.gui.widget.EnergyWidget;
+import com.enderio.machines.client.gui.widget.FluidStackStaticWidget;
+import com.enderio.machines.client.gui.widget.ProgressWidget;
+import com.enderio.machines.common.menu.PaintingMachineMenu;
+import com.enderio.machines.common.menu.XPVacuumMenu;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.client.gui.components.ImageButton;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Inventory;
+
+public class PaintingMachineScreen extends EIOScreen<PaintingMachineMenu> {
+
+    private static final ResourceLocation PAINTING_MACHINE_BG = EnderIO.loc("textures/gui/painting_machine.png");
+
+    public PaintingMachineScreen(PaintingMachineMenu pMenu, Inventory pPlayerInventory, Component pTitle) {
+        super(pMenu, pPlayerInventory, pTitle, true);
+        this.inventoryLabelY = this.imageHeight - 106;
+    }
+
+    @Override
+    protected void init() {
+        super.init();
+        addRenderableOnly(new EnergyWidget(this, getMenu().getBlockEntity()::getEnergyStorage, 16 + leftPos, 14 + topPos, 9, 42));
```

Comment:
Make the painter use the new capacitor tooltip energy widget

```suggestion
         addRenderableOnly(new CapacitorEnergyWidget(this, getMenu().getBlockEntity()::getEnergyStorage, getMenu().getBlockEntity()::isCapacitorInstalled, 16 + leftPos, 14 + topPos, 9, 42));
```